HW0/P5/P5.txt

```diff
@@ -1 +1,21 @@
 # Your answers here.
+
+#1. It would take the cashier 255 seconds, because it takes 255 times to sum up all the 256 numbers if add two numbers at a time, and it takes one second to add two numbers.
+
+#2. They can verify the total sum faster than the original cashier. They can do this in 34 seconds. The fastest way is for each cashier to verify the same number of bags: 256/8 = 32, it takes all of them 31 seconds. Then it takes 3 seconds to add the 8 numbers. In total, it takes 34 seconds.
+
+#3. If the store has an infinite number of employees, it takes 8 seconds to verify the sum. First, all 256 bags are verified using 1 second. Then add two numbers at a time, there are 128 counts using another 1 second. Continue this process, it takes 8 steps to add all numbers, which uses 8 seconds.
+
+#4. Suppose the time to count total is x, for these N bags, we have 2^(x) = N if adding two bags at a time. Thus the time to count is log2(N). 
```

Comment:
And for the plot, because the time infinite workers used should be ceiling of log2(N) instead of log2(N), the line should be discrete.